common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -571,3 +571,24 @@ def test_special_exams_menu_a11y(self):
             include=['section.edit-settings-timed-examination']
         )
         self.course_outline.a11y_audit.check_for_accessibility_errors()
+
+
+class StudioUploadCourseImageTest(StudioCourseTest):
+
+    """
+    Class to test course image upload.
+    """
+
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(StudioUploadCourseImageTest, self).setUp()
+        self.settings_page = SettingsPage(self.browser, self.course_info['org'], self.course_info['number'],
+                                          self.course_info['run'])
+
+    def test_upload_course_image(self):
```

Comment:
Thanks for adding this bok choy test for the existing course image. Please also add coverage for your two new images.

If this test duplicates what is covered in the lettuce test course-settings.feature, please delete that old test.